src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.cs

```diff
@@ -29,6 +29,12 @@ internal sealed partial class PortableThreadPool
         private const int CpuUtilizationHigh = 95;
         private const int CpuUtilizationLow = 80;
 
+#if CORECLR
+#pragma warning disable CA1823
+        private static readonly bool s_initialized = ThreadPool.EnsureConfigInitialized();
```

Comment:
What is the reason to use an unused field here? 
Where is this used?

Reply:
This is used to ensure that the threadpool config is initialized. It is the minimal change to preserve the existing initialization flow.

As mentioned above, there are more cleanups that can be done here, not part of this PR.

Reply:
Okay. I was thinking of a static constructor, but of course it works that way too.

Reply:
Adding regular static constructor would add static constructor triggers to all static methods that may introduce performance regressions.